task/jq/0.1/jq.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: jq
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.56.1"
+    tekton.dev/categories: Developer Tools
+    tekton.dev/tags: jq
+    tekton.dev/displayName: "JQ"
+    tekton.dev/platforms: "linux/amd64"
+spec:
+  description: >-
+    Task built for using the JQ utility in pipelines.
+  workspaces:
+    - name: source
+      description: A workspace that contains the file which needs to be altered. It can take the form of a persistent volume (PV and PVC) or configmap.
+      optional: true
+  params:
+    - name: options
+      type: string
+      description: JQ options that will be used. If none is provided, the default value '' will be used."
+      default: "-M"
+    - name: filter
+      type: string
+      description: "JQ filter that will be executed. If none is provided, the the filter '.' will be executed by default."
+      default: "."
+    - name: image
+      type: string
+      description: "The jq image to use."
+      default: quay.io/jpmaida1/jq@sha256:f90645c017ba8b1a2fff44309231cb7f8fd3a441690e17c15a53e60997f309c0
```

Comment:
is there any public image available for jq ?

Reply:
The image is located in a public repo

Reply:
@jpmaida, I meant any maintained image available? other than present under your account?

Reply:
I've found the image [](https://github.com/jqlang/jq/pkgs/container/jq). It seems to be maintained by the original JQ's author. However some aspects must be considered, they are:
- I couldn't find or there isn't a way to inspect the images and see its Dockerfile
- There is no assurance that container best practices were used
- Quay brings by default a security analysis, github doesn't

What do we do?